src/main/java/com/radixdlt/atommodel/validators/RegisteredValidatorParticle.java

```diff
@@ -27,32 +28,36 @@
 import java.util.Set;
 
 @SerializerId2("radix.particles.registered_validator")
-public class RegisteredValidatorParticle extends Particle {
+public final class RegisteredValidatorParticle extends Particle {
 	@JsonProperty("address")
 	@DsonOutput(DsonOutput.Output.ALL)
-	private RadixAddress address;
+	private final RadixAddress address;
 
 	@JsonProperty("allowedDelegators")
 	@DsonOutput(DsonOutput.Output.ALL)
-	private Set<RadixAddress> allowedDelegators;
+	private final ImmutableSet<RadixAddress> allowedDelegators;

```

Comment:
This is the relevant change here for this commit.  Previously the Set<...> was filled in with a HashSet, which resulted in the ordering being undefined.  Using an ImmutableSet ensures a consistent ordering.